docs/rules/new-line-before-expect.md

```diff
@@ -0,0 +1,47 @@
+# Enforce new line before expect inside a suite
+
+This rule enforces that there is an empty line before `expect`.
+
+## Rule details
+
+This rule triggers a **warning** (is set to **1** by default) whenever there is no new line before expect.
+
+The following patterns are considered warnings:
+
+```js
+describe("", function() {
+  it("", function() {
+    expect(1).toBe(1);
```

Comment:
I think if the expect is the first line on the `it` block it shouldn't be a warning